nf_core/lint/nextflow_config.py

```diff
@@ -262,6 +268,44 @@ def nextflow_config(self):
                 )
             )
 
+    # Check if custom profile params are set correctly
+    if self.nf_config.get("params.custom_config_version", "").strip("'") == "master":
+        passed.append("Config ``params.custom_config_version`` is set to ``master``")
+    else:
+        failed.append("Config ``params.custom_config_version`` is not set to ``master``")
+
+    custom_config_base = "https://raw.githubusercontent.com/nf-core/configs/" + self.nf_config.get(
```

Comment:
On my phone so off the top of my head, but this should be a static string rather than taking the interpolated value, no? Or do we just pick up the final version? 🤔

Reply:
I thought so at first as well, but since we are comparing against the value in the parsed config file, we need to interpolate the value.